### PR TITLE
Add release notes for CLI v1.3.0

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

Add release notes for CLI v1.3.0 release

## Motivation and Context
New release, new notes

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,20 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 
 <?! Include "../../shared/chocolatey-component-dependencies-link.txt" /?>
 
+## [1.3.0](https://github.com/chocolatey/choco/milestone/59?closed=1) (February 15, 2023)
+
+### Deprecated Features
+
+- Deprecate WebPI alternative source - see [#2958](https://github.com/chocolatey/choco/issues/2958)
+- Deprecate usage of direct path to nupkg or nuspec in choco install command - see [#2981](https://github.com/chocolatey/choco/issues/2981)
```
Comment (gep13) [inline input/en-us/choco/release-notes.md:26]:
```suggestion
- Deprecate usage of direct path to nupkg or nuspec in `choco install` command - see [#2981](https://github.com/chocolatey/choco/issues/2981)
```

File: input/en-us/choco/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,20 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+- Deprecate the --remove option in the apikey command - see [#2983](https://github.com/chocolatey/choco/issues/2983)
```
Comment (gep13) [inline input/en-us/choco/release-notes.md:27]:
```suggestion
- Deprecate the `--remove` option in the apikey command - see [#2983](https://github.com/chocolatey/choco/issues/2983)
```

File: input/en-us/choco/release-notes.md [hunk 3]
```diff
@@ -18,6 +18,20 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+- Deprecate choco config list including output from features/sources/apikeys etc - see [#2985](https://github.com/chocolatey/choco/issues/2985)
```
Comment (gep13) [inline input/en-us/choco/release-notes.md:28]:
```suggestion
- Deprecate `choco config list` including output from features/sources/apikeys etc - see [#2985](https://github.com/chocolatey/choco/issues/2985)
```

File: input/en-us/choco/release-notes.md [hunk 4]
```diff
@@ -18,6 +18,20 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+- Deprecate unnecessary public methods on CygwinService - see [#3026](https://github.com/chocolatey/choco/issues/3026)
+
+### Bug Fixes
+
+- File not found error when using --source=cygwin to install packages - see [#2953](https://github.com/chocolatey/choco/issues/2953)
```
Comment (gep13) [inline input/en-us/choco/release-notes.md:33]:
```suggestion
- File not found error when using `--source=cygwin` to install packages - see [#2953](https://github.com/chocolatey/choco/issues/2953)
```

**Review by gep13 — CHANGES_REQUESTED** [review]
Just a couple of minor things. The GitHub issues don't render markdown code fragments, but it makes sense to put them here in the release notes that go out on the site.

**Review by pauby — CHANGES_REQUESTED** [review]
Can we add a full stop to the end of each line?

**Comment by vexx32** [comment]
> Can we add a full stop to the end of each line?

Is that something GitReleaseManager can do if we ask nicely, for future reference?

**Comment by vexx32** [comment]
@gep13 @pauby I've updated the release notes accordingly :)

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
@vexx32 Approved. I'll leave you to merge when you need to.

**Pull request merged**